packages/salesforcedx-utils-vscode/src/cli/commandBuilder.ts

```diff
@@ -58,9 +58,11 @@ export class CommandBuilder {
     return this;
   }
 
-  public withJson(): CommandBuilder {
+  public withJson(logLevel: boolean = true): CommandBuilder {
     this.args.push('--json');
-    this.args.push('--loglevel', 'fatal');
+    if (logLevel) {
+      this.args.push('--loglevel', 'fatal');
+    }
```

Comment:
Since we are now using the `SF_LOG_LEVEL` environment variable instead of the deprecated `--loglevel` flag, is this  change necessary?  Or is it ok to just take out the `--loglevel` flag entirely?

Reply:
that flag is still used in other commands, so I'd keep it until the very last moment when we actually change from sfdx to sf or when no other commands are left to be transitioned